roles/oneagent/tests/constants.py

```diff
@@ -0,0 +1,80 @@
+from enum import Enum
+from pathlib import Path
+import tempfile
+
+COLLECTION_NAMESPACE = "dynatrace"
+COLLECTION_NAME = "oneagent"
+ROLE_NAME = "oneagent"
+
+# TODO: remove in tests refactor
```

Comment:
Is this an old `TODO`? If so, is there any story related to it? In such ase it should be removed.